cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1405,20 +1405,6 @@ def test_json_responses(self):
         json_response = json.loads(resp.content)
         self.validate_course_xblock_info(json_response, course_outline=True)
 
-    def test_chapter_entrance_exam_xblock_info(self):
```

Comment:
You should instead test the new logic you added about deletable, addable, draggable, explanatory_text, and is_visible (please also test default behavior for when the xblock is not related to an entrance exam.